taworks/taform/views.py

```diff
@@ -474,6 +474,7 @@ def export_rankings():
     df.drop(['course_subject', 'course_id', 'section', 'course_name', 'c_id', 's_id', 'student_id', 
         'first_name', 'last_name', 'email'], axis = 1, inplace = True)
     df = df[['course_unit', 'student_unit', 'instructor_preference', 'preference']]
+    df[['instructor_preference']] = df[['instructor_preference']].fillna(0).astype(int)
```

Comment:
since 0 actually means not a match in our model. 
I think keeping it as NaN is fine or something like -1 (it doesn't map to anything).

    ratings = ((1,'1-Most Preferred'),(2,'2'),(3,'3'),(4,'4'),(5,'5-Least Preferred'),(0,'0-Not a Match'))